content/en/api_catalog/owners_and_tags.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Establishing Owners and Tagging Endpoints
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+<!-- image TKTK -->
+
+## Establishing owners
+
+The **TEAM** column on the Explorer page and each endpoint details page shows the name of the team that owns the API that the endpoint is associated with.
+
+To assign ownership to an endpoint:
+
+1. From the **Explorer** page, select the endpoint(s) that need owners.
+2. Click **Register Endpoints**.
+3. Search for then select the teams that should own the endpoint.
```

Comment:
@ddOfir - This menu only seems to show APIs to group by? How do we differentiate between teams and APIs when registering endpoints?

Reply:
Yes you indeed choose an API, and the API has an owner...

maybe the right wording here would be "search for then select an API you want to assign endpoints to"

And then maybe we can somewhere add that and API equals an owner

Reply:
I think i need to totaly refactor this section. It will take me some time (day or 2) is it possible to wait until i re-write this section?

Reply:
You bet! Sounds good. What you're saying makes sense to me. Based on how it was originally written, I assumed there would be a distinct Owner option from the API.

Reply:
yes...

I and @barashkenazy changed this part, you can see it in the doc (tagged you)